packages/react/src/Suspense.spec.tsx

```diff
@@ -57,3 +57,37 @@ describe('Suspense', () => {
     expect(screen.queryByText(TEXT)).toBeInTheDocument()
   })
 })
+
+const SuspendDuring100msToShowTEXTInSuspense = withSuspense(() => <Suspend during={MS_100} toShow={TEXT} />, {
+  fallback: FALLBACK,
+})
+const SuspendDuring100msToShowTEXTInSuspenseCSROnly = withSuspense.CSROnly(
```

Comment:
It's minor!

How about matching it with the actual component interface?
```suggestion
SuspendDuring100msToShowTEXTInSuspense.CSROnly = withSuspense.CSROnly(
```

Reply:
Cool

Reply:
When I apply this suggestion, I realized we need to type complexly. I'll try this after Pull Request